### ISSUE TITLE
Comments: Tiled galleries do not upload

### ISSUE THREAD

**Issue by yaelirub** [Automattic/wp-calypso#45138]
#### Steps to reproduce

1. On the web for a WP.com or Jetpack site, comment on a post.
2. Add a paragraph.
3. Add a tiled gallery block with several images.
4.Add a few more tiled gallery blocks with various settings.
5. Add another paragraph.
6.Publish the comment

#### What I expected
I expected the tiled gallery block to upload the images and show in the comment 

#### What happened instead
The tiled gallery images keeps flashing like they are going to upload but nothing happens

#### Screenshot / Video
<img width="820" alt="Screen Shot 2020-08-21 at 6 20 52 PM" src="https://user-images.githubusercontent.com/1335657/90945669-28cba880-e3db-11ea-972e-300e68a1aa80.png">

**Comment by davipontesblog** [comment]
Part of the Block Audit milestone: https://github.com/Automattic/wp-calypso/milestone/320
As discussed with @4thhubbard, leaving it for future consideration.

**Comment by simison** [comment]
I don't really understand how to replicate this. :-) 

It's not possible to write blocks in comments on .com:

<img width="1054" alt="Screenshot 2021-05-14 at 16 08 44" src="https://user-images.githubusercontent.com/87168/118275383-06307100-b4cf-11eb-8e7a-86924fd0b669.png">

Then I remembered we used to test Gutenberg for comments in WordPress.com reader, so perhaps that's where? It doesn't seem to be enabled currently though:

<img width="800" alt="image" src="https://user-images.githubusercontent.com/87168/118275503-2a8c4d80-b4cf-11eb-8a88-a873ce8710f0.png">

Meanwhile I lowered this to normal priority, and logged for View team for triage since this is blocks related.
